service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/C21OrderEmailContentProvider.java

```diff
@@ -25,31 +26,37 @@
 public class C21OrderEmailContentProvider extends AbstractEmailContentProvider {
 
     private final LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration;
+    private final CafcassLookupConfiguration cafcassLookupConfiguration;
     private final ObjectMapper objectMapper;
 
     protected C21OrderEmailContentProvider(@Value("${ccd.ui.base.url}")String uiBaseUrl,
                                            ObjectMapper objectMapper,
                                            HearingBookingService hearingBookingService,
                                            LocalAuthorityNameLookupConfiguration
                                                localAuthorityNameLookupConfiguration,
-                                           DateFormatterService dateFormatterService) {
+                                           DateFormatterService dateFormatterService,
+                                           CafcassLookupConfiguration cafcassLookupConfiguration) {
         super(uiBaseUrl, dateFormatterService, hearingBookingService);
         this.objectMapper = objectMapper;
         this.localAuthorityNameLookupConfiguration = localAuthorityNameLookupConfiguration;
+        this.cafcassLookupConfiguration = cafcassLookupConfiguration;
     }
 
-    public Map<String, Object> buildC21OrderNotification(final CaseDetails caseDetails,
-                                                         final String localAuthorityCode) {
-        // Validation within our frontend ensures that the following data is present
-        CaseData caseData = objectMapper.convertValue(caseDetails.getData(), CaseData.class);
-        final String subjectLine = buildSubjectLine(caseData);
-        return ImmutableMap.of(
-            "subjectLine", subjectLine,
-            "localAuthorityOrCafcass", localAuthorityNameLookupConfiguration.getLocalAuthorityName(localAuthorityCode),
-            "hearingDetailsCallout", buildSubjectLineWithHearingBookingDateSuffix(subjectLine, caseData),
-            "reference", String.valueOf(caseDetails.getId()),
-            "caseUrl", uiBaseUrl + "/case/" + JURISDICTION + "/" + CASE_TYPE + "/" + caseDetails.getId()
-        );
+    public Map<String, Object> buildC21OrderNotificationParametersForCafcass(final CaseDetails caseDetails,
+                                                                             final String localAuthorityCode) {
+        return ImmutableMap.<String, Object>builder()
+            .putAll(commonC21NotificationParameters(caseDetails))
+            .put("localAuthorityOrCafcass", cafcassLookupConfiguration.getCafcass(localAuthorityCode).getName())
```

Comment:
That will produce "Cafcass Cymru" are we 100% sure this is what business wants? Can's see this in the AC.

Reply:
Hi @akomisarek, we have confirmed this is the case from Jay and Mia.